scanpy/neighbors/__init__.py

```diff
@@ -92,8 +93,17 @@ def neighbors(
         random_state=random_state)
     adata.uns['neighbors'] = {}
     adata.uns['neighbors']['params'] = {'n_neighbors': n_neighbors, 'method': method}
+    adata.uns['neighbors']['params']['metric'] = metric
+    if metric_kwds:
+        adata.uns['neighbors']['params']['metric_kwds'] = metric_kwds
+    if use_rep is not None:
+        adata.uns['neighbors']['params']['use_rep'] = use_rep
+    if n_pcs is not None:
+        adata.uns['neighbors']['params']['n_pcs'] = n_pcs
     adata.uns['neighbors']['distances'] = neighbors.distances
     adata.uns['neighbors']['connectivities'] = neighbors.connectivities
+    if neighbors.rp_forest is not None:
```

Comment:
But, as mentioned, I'd not at the `rp_forest` object at this stage. Makes sense? Can you remove it?